ocpi-tariffs/src/lib.rs

```diff
@@ -48,6 +59,8 @@ impl fmt::Display for Error {
         let display = match self {
             Self::NoValidTariff => "No valid tariff has been found in the list of provided tariffs",
             Self::NumericOverflow => "A numeric overflow occurred during tariff calculation",
+            Self::TimeZoneMissing => "No time zone could be found in the session information",
+            Self::TimeZoneInvalid => "The time zone specified is invalid",
```

Comment:
:wrench: the above documentation suggests that this invalid time zone could be in the CDR as well, do we want to hint at that in the error message? _specified_, might not be clear enough.